src/common/containers/Profile.js

```diff
@@ -1,28 +1,102 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { object, func } from 'prop-types';
+import { withRouter } from 'react-router-dom';
 import Helmet from 'react-helmet';
 import { connect } from 'react-redux';
-import { withRouter } from 'react-router';
+import { withStyles } from '@material-ui/core/styles';
 import Paper from '@material-ui/core/Paper';
 
+import RecentCardsCarousel from '../components/cards/RecentCardsCarousel';
+import Title from '../components/Title';
+import * as collectionActions from '../actions/collection';
+
 export function mapStateToProps(state) {
   return {
-    loggedIn: state.global.user !== null
+    user: state.global.user
   };
 }
 
-const About = function (props) {
-  return (
-    <div style={{margin: '48px 72px'}}>
-      <Helmet title="Profile"/>
+export function mapDispatchToProps(dispatch) {
+  return {
+    onOpenForEditing: (card) => {
+      dispatch(collectionActions.openForEditing(card));
+    }
+  };
+}
 
-      <div style={{display: 'flex', justifyContent: 'stretch'}}>
-        <Paper style={{padding: '5px 20px'}}>
-          <h2>Profile</h2>
-          <p>TODO: Stuff goes here.</p>
-        </Paper>
-      </div>
-    </div>
-  );
+const styles = {
+  container: {
+    margin: 20
+  },
+  paperContainer: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  gridItem: {
+    width: 'calc((100% / 3) - (40px / 3))',
+    height: 600
```

Comment:
Does this need to be that big? It looks bad on my screen.
![image](https://user-images.githubusercontent.com/415965/44635000-4a613900-a955-11e8-833e-b8d955167cd3.png)

Reply:
This is pretty temporary. It's definitely can be smaller in the future